.tekton/kf-notebook-controller-odh-io-main-on-push.yaml

```diff
@@ -0,0 +1,591 @@
+# yamllint disable-file
+# This file is autogenerated by ci/konflux/generate_component_build_pipelines.py
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/opendatahub-io/notebooks?rev={{revision}}
```

Comment:
should this be pointing to this repo ?
the configuration seems to have been all on this repo, is there some reason for this to point to notebooks repo?